.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
```

Comment:
Comments up here seem not relevant. I assume you customized the python example, but this is a node project :) Suggest just deleting all 4 of these lines.

Reply:
Haha, yes good call.